web/src/client/http.js

```diff
@@ -106,6 +106,15 @@ class HTTPClient {
     return response;
   }
 
+  /**
+   * @param {string} url - Endpoint URL (e.g., "/l10n/config").
+   * @return {Promise<Response>} Server response.
+   */
+  async getRaw(url) {
```

Comment:
Instead of extending the API (which is expected to be used only here), you could just use fetch. WDYT?

Reply:
In fact originally it was download, and we (@dgdavid and me) were evaluating all the options you mentioned even adding an option for specifying the content type as a parameter for the get method but didn't liked any of them at all.

By now it is only used in this case so, just using fetch should be enough.